src/Apps/Artwork/Components/PrivateArtwork/PrivateArtworkAboutArtist.tsx

```diff
@@ -48,13 +48,7 @@ export const PrivateArtworkAboutArtist: React.FC<PrivateArtworkAboutArtistProps>
             }
           }
           name
-          partnerArtists {
-            biography
-          }
-          partnerBiographyBlurb: biographyBlurb(
-            format: HTML
-            partnerBio: true
-          ) {
+          partnerBiographyBlurb {
```

Comment:
@jpotts244 - don't we need to update the code somewhere (vs just the graphql query?)

Reply:
ahhhh I see 😄

Reply:
Dont think so!! I think yours was the same naming, nice clean diff ✅